### ISSUE TITLE
Add file sizes and dimensions to metadata

### ISSUE THREAD

**Issue by jeroenbranje** [ASCS-eV/envited-x-data-space#375]
# Problem statement
For creating the token metadata we have a hardcoded mime type, file sizes and dimensions

# Definition of done
The file information should be taken from the file that's extracted

**Comment by jdsika** [comment]
```
import fs from "fs/promises";
import { statSync } from "fs";

async function getFileSize(filePath: string): Promise<number> {
    try {
        // Check if the path is a remote URL
        if (filePath.startsWith("http")) {
            const response = await fetch(filePath, { method: "HEAD" });
            if (!response.ok) {
                throw new Error(`Failed to fetch file: ${response.statusText}`);
            }
            const contentLength = response.headers.get("content-length");
            if (!contentLength) {
                throw new Error("Content length not provided by server.");
            }
            return parseInt(contentLength, 10);
        } else {
            // Handle local file
            const stats = statSync(filePath);
            return stats.size;
        }
    } catch (error) {
        console.error(`Error calculating size for file ${filePath}: ${error.message}`);
        return 0; // Return 0 as fallback
    }
}

async function calculateFileSizes() {
    const artifactUri = "path_or_url_to_artifactUri"; // Replace with actual path/URL
    const externalUri = "path_or_url_to_externalUri"; // Replace with actual path/URL
    const displayUri = "path_or_url_to_displayUri"; // Replace with actual path/URL

    try {
        const [artifactSize, externalSize, displaySize] = await Promise.all([
            getFileSize(artifactUri),
            getFileSize(externalUri),
            getFileSize(displayUri),
        ]);

        console.log(`File Sizes (in bytes):`);
        console.log(`Artifact URI: ${artifactSize}`);
        console.log(`External URI: ${externalSize}`);
        console.log(`Display URI: ${displaySize}`);
    } catch (error) {
        console.error("Error calculating file sizes:", error);
    }
}

// Run the calculation
calculateFileSizes();

```

**Comment by jdsika** [comment]
The calculation can be done after the files are extracted and saved to the buckets #376

**Comment by jdsika** [comment]
resolved by #374
